web/pdf_viewer.js

```diff
@@ -82,6 +82,8 @@ function isValidAnnotationEditorMode(mode) {
  * @typedef {Object} PDFViewerOptions
  * @property {HTMLDivElement} container - The container for the viewer element.
  * @property {HTMLDivElement} [viewer] - The viewer element.
+ * @property {HTMLDivElement} [hiddenElement] - The hidden element used to check
```

Comment:
Here, and elsewhere, should we perhaps name this e.g. `copyElement` instead to more clearly indicate its actual use?